apps/aecore/src/aec_conductor.erl

```diff
@@ -409,8 +425,9 @@ dispatch_worker(Tag, Fun, State) ->
         false ->
             Wrapper = wrap_worker_fun(Fun),
             {Pid, Ref} = spawn_monitor(Wrapper),
+            {ok, TRef} = timer:exit_after(_TODO=200, Pid, shutdown),
```

Comment:
I was not aware of this function. Nice!

However, I've had some problems with using timer.erl and prefer using erlang:set_timer/cancel_timer which is using the timer wheel in the vm rather than another gen_server. 

On the other hand, we are not using that many timers, so it is unlikely that we would see problems with this.

Just some ramblings. Do as you please.

Reply:
I agree with you on timer functions in erlang module to be preferred over those in timer module.
Though, as you pointed out, we are note using many timers. And the convenience of timer:exit_after made me opt for this.